config/application.rb

```diff
@@ -54,6 +54,43 @@
   Bundler.require(*bundler_groups)
 end
 
+def with_plugin_guard(&block)
+  begin
+    block.call
```

Comment:
Very minor but you could avoid the `&block` parameter altogether and use `yield`

Reply:
I actually changed it from `yield` to `&block` before committing. 😛 I find it useful to have an explicit block reference in a method signature - you can see if given method uses a block without reading its code.